projects/canopy/src/styles/mixins.scss

```diff
@@ -13,13 +13,72 @@
   box-shadow: 0 0 0 0.25rem $bg-color;
 }
 
-@mixin lg-background-focus(
-  $bg-color: var(--link-focus-background-color),
-  $color: var(--link-focus-color)
+@mixin lg-link(
+  $default-color: var(--link-color),
+  $hover-color: var(--link-hover-color),
+  $visited-color: var(--link-visited-color),
+  $active-color: var(--link-active-color),
+  $active-bg-color: var(--link-active-bg-color),
+  $focus-color: var(--link-focus-color),
+  $focus-bg-color: var(--link-focus-bg-color)
 ) {
-  background: $bg-color;
-  color: $color;
-  outline: var(--link-focus-outline) solid $bg-color;
+  $box-shadow-inset-width: -0.063rem;
+
+  color: $default-color;
+  text-decoration: none;
+  border-bottom: 0.125rem solid $default-color;
+  padding: 0 0.125rem;
+
+  &:hover {
+    color: $hover-color;
+    border-bottom: 0;
+    box-shadow: inset 0 $box-shadow-inset-width 0 0 $hover-color,
+      0 0.188rem 0 0 $hover-color;
+  }
+
+  &:visited {
+    color: $visited-color;
+    border-color: $visited-color;
+  }
+
+  &:active {
+    background-color: $active-bg-color;
+    border-bottom-color: $active-color;
+    color: $active-color;
+    outline: 0;
+  }
+
+  &:focus {
+    background-color: $focus-bg-color;
+    border-bottom: 0;
+    color: $focus-color;
+    outline: 0.063rem solid $focus-bg-color;
+    outline-offset: 0;
+
+    &:hover {
+      box-shadow: none;
+    }
+  }
+}
+
+@mixin lg-revert-link-style() {
```

Comment:
I think it would be useful to provide a utility class for apps that don't use or import the SASS files. E.g. in our project we only import the main **canopy.css** file. Perhaps something like this?

```
.lg-revert-link-style {
  @include lg-revert-link-style;
}
```